docs/source/neo/SmartContract/smartcontracts.rst

```diff
@@ -456,7 +456,7 @@ Hopefully this is enough to get you started with building and testing your Smart
 Importing a Smart Contract
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Smart Contract importing is somewhat similar to the ``build .. test`` notation, though you do not need to send any parameters along with.  The format is ``import contract path/to/sample2.avm {input_params} {return_type} {needs_storage} {needs_dynamic_invoke}``.  After running this command, if everything goes ok you will be prompted to add some metadata about the contract.  Once that is complete, you will then have the choice to actually deploy this Smart Contract on the network.  Beware that doing so will cost you some Gas!
+Smart Contract importing is somewhat similar to the ``build .. test`` notation, though you do not need to send any parameters along with it.  The format is ``import contract path/to/sample2.avm {input_params} {return_type} {needs_storage} {needs_dynamic_invoke}``.  After running this command, if everything goes ok you will be prompted to add some metadata about the contract.  Once that is complete, you will then have the choice to actually deploy this Smart Contract on the network.  Beware that doing so will cost you some Gas!
```

Comment:
note that I added 'it' at the end of the first sentence. Not sure why it doesn't recognise it.